juju/juju
# [JUJU-1821] Fixed test-controller-test-unregister-test

This PR fixes the test-controller-test-unregister-test tests, the original test does not consider that `controllers.yaml` file may not exist.

The test is now rewritten using the fake `controllers.yaml` file created in the test folder.
The test is moved from the controller suite to the cli suite.

## Checklist


## QA steps

*Commands to run to verify that the change works.*

```sh
cd tests
./main.sh -v cli run_unregister
```
